runtime/gc_base/StringTable.cpp

```diff
@@ -556,9 +594,19 @@ j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA s
 					break;
 				}
 			}
+			isASCIIorLatin1 = isASCII;
+			if (!isASCII && J2SE_VERSION(vm) >= J2SE_V17) {
```

Comment:
Please bracket the second test (here and everywhere else).

Reply:
Added the bracket.